### ISSUE TITLE
AsyncConnection ZeroBuffers issue for MacOS on .NET Core 3.1

### ISSUE THREAD

**Issue by mdingjax** [aerospike/aerospike-client-csharp#48]
Conditions on setting the "ZeroBuffers" doesn't align with latest .NET Core 3.1. It misses the condition where the new repalcement value for MacOSX is Unix. See MSDN here: https://docs.microsoft.com/en-us/dotnet/api/system.platformid?view=netcore-3.1

**Comment by BrianNichols** [comment]
In the next release ZeroBuffers will be changed to:

```
private readonly static bool ZeroBuffers = !(
	Environment.OSVersion.Platform == PlatformID.Unix ||
	Environment.OSVersion.Platform == PlatformID.MacOSX);
```

**Comment by BrianNichols** [comment]
Fixed in C# client release 3.9.3:

https://www.aerospike.com/download/client/csharp/3.9.3/
